.changeset/fresh-comics-carry.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
are you planning to land these as two separate commits? That would be the tidier way IMHO

Reply:
That's fair, I can definitely split it out into another PR (I did it like this initially because I needed a pre-release with both changes)

Reply:
if you can split this PR in two dedicated commit, that's already helpfu

Reply:
It won't make much difference outside of PR review, since we squash all the commits from a PR into a single commit when merging. So unless this will aid review, I wouldn't bother.

Reply:
oh right, those will automatically get squashed :/. nvm then